File: CS337/Lab1/odd.go
package main
import (
	"fmt"
)

func isOdd(n int) bool {
	if n % 2 == 1 {
		return true
	} else {
		return false
	}
}

func main() {
	for i := -5; i <= 5; i++ {
		fmt.Println(i, isOdd(i))
	}
}

File: CS337/Lab1/Odd.java
public class Odd {
    public static void main(String[] args) {
        Odd odd = new Odd();
        for (int i = -5; i <= 5; i++) {
            System.out.println(i + " " + odd.isOdd(i));
        }
    }

    public boolean isOdd(int n) {
        if (n % 2 == 1) {
            return true;
        } else {
            return false;
        }
    }
}